Gemfile

```diff
@@ -7,7 +7,8 @@ ruby '2.3.1'
 #=== CORE =====================================================================
 gem 'acts-as-taggable-on', git: 'https://github.com/Fodoj/acts-as-taggable-on', ref: 'rails-5.2'
 gem 'carrierwave', '~> 1.2'
-gem 'devise', '~> 4.6'
+gem 'devise', '~> 4.7', '>= 4.7.1'
```

Comment:
could be: 

`gem 'devise', '~> 4.7.1'`

Reply:
Thanks. Fixed